marklogic-client-api-functionaltests/src/test/java/com/marklogic/client/functionaltest/ConnectedRESTQA.java

```diff
@@ -1018,101 +1022,18 @@ public static void tearDownJavaRESTServer(String dbName, String[] fNames, String
 		logTestMessages(" Ending TESTCASE TEARDOWN ", beforeTeardown);
 	}
 
-	// Setting up AppServices configurations setting up database properties whose value is string
-	public static void setDatabaseProperties(String dbName, String prop, String propValue) throws IOException {
-		String resGet = null;
-		JsonNode jnode = null;
-		Response responsePut = null;
-		OkHttpClient client = createManageAdminClient("admin", "admin");
-		try {
-			String getrequest = new String("http://" + host_name + ":" + admin_port + "/manage/v2/databases/" + dbName
-					+ "/properties?format=json");
-			Request request = new Request.Builder()
-					.header("Content-type", "application/json")
-					.url(getrequest)
-					.build();
-			Response response1 = client.newCall(request).execute();
-			if (response1.code() == ML_RES_OK) {
-				resGet = response1.body().string();
-				System.out.println("Response from Get is " + resGet);
-			}
-			if (resGet != null && !resGet.isEmpty())
-				jnode = new ObjectMapper().readTree(resGet);
-			else throw new Exception("Unexpected error " + response1);
-
-			if (!jnode.isNull()) {
-				((ObjectNode) jnode).put(prop, propValue);
-				String putUrl = new String("http://" + host_name + ":" + admin_port + "/manage/v2/databases/" + dbName
-						+ "/properties?format=json");
-
-				String putProps = jnode.toString();
-				Request requestPut = new Request.Builder()
-						.header("Content-type", "application/json")
-						.url(putUrl)
-						.put(RequestBody.create(putProps, MediaType.parse("application/json")))
-						.build();
-				responsePut = client.newCall(requestPut).execute();
-
-				if (responsePut.code() == ML_RES_CHANGED) {
-					System.out.println("Database " + dbName + ". property " + prop +" has been updated with " + propValue);
-				}
-			} else {
-				System.out.println("REST call for database properties update has issues");
-				System.out.println(responsePut.toString());
-			}
-		} catch (Exception e) {
-			e.printStackTrace();
-		} finally {
-			client = null;
-		}
+	public static void setDatabaseProperties(String dbName, String prop, String propValue) {
+		ObjectNode properties = new ObjectMapper().createObjectNode();
```

Comment:
ConnectedRESTQA has a lot of duplicated code on the left for making calls to the Manage API. Going to gradually convert those all over to use ManageClient from the ml-app-deployer. Will get rid of hundreds of lines of duplicated code.